examples/3DoFTest.cpp

```diff
@@ -0,0 +1,86 @@
+// BSD 3-Clause License
```

Comment:
I use Eigen here with our stuff for the first time, and in theory this would be 3DOF specific, not sure if that changes things or if I should still pull this back